tests/test_target.py

```diff
@@ -37,23 +36,57 @@ def build_targets():
 
     targets = []
 
+    if target_configs.get('AndroidTarget') is not None:
+        print('> Android targets:')
+        for entry in target_configs['AndroidTarget'].values():
+            pp(entry)
+            a_target = AndroidTarget(
+                connection_settings=entry['connection_settings'],
+                conn_cls=lambda **kwargs: AdbConnection(adb_as_root=True, **kwargs),
```

Comment:
Not an issue per say here, but FYI `functools.partial()` does the same as that lambda, somewhat better in minor ways

Reply:
Will keep it as is.